cmd/release-controller/main.go

```diff
@@ -161,6 +163,8 @@ func main() {
 
 	flagset.StringVar(&opt.AuthenticationMessage, "authentication-message", opt.AuthenticationMessage, "HTML formatted string to display a registry authentication message")
 
+	flagset.StringVar(&opt.Registry, "registry", opt.Registry, "Specify an alternate image registry to reference.  This option should be used when running the controller on a different cluster.")
```

Comment:
I don't understand what the text is telling me to do.  Describe how this flag is used, not when it is used.

I.e. "change the registry that the artifact server will use to retrieve release images when located on remote clusters" etc

Reply:
Updated message to:
```
Specify the registry, that the artifact server will use, to retrieve release images when located on remote clusters
```